index.html

```diff
@@ -0,0 +1,219 @@
+                  <div><p class="back-end">Back End Dev</p></div>
+                  <div><img src="images/Counter.png" alt=""></div>
+                  <div><p class="back-end">2015</p></div>
+              </div>
+              <div class="access-project">
+                  <p>A daily selection of privately personalized reads. no accounts or sign-ups required</p>
+              </div>
+              <div class="programming-lang">
+                <div class="lang"><p>html</p></div>
+                <div class="lang"><p>css</p></div>
+                <div class="lang"><p>javascript</p></div>
+              </div>
+              <div class="project-btn"><button>See project</button></div>
+          </div>
+      </div>
+  </section>
+  <section>
+      <div class="main-container">
+        <div class="project-container">
+          <div class="developer"><img src="images/Snapshoot Portfolio.png" alt=""></div>
+          <div class="project-review">
+              <label>Tonic</label>
+          </div>
+          <div class="backend-dev">
+              <div><p class="canopy">Canopy</p></div>
+              <div><img src="images/Counter.png" alt=""></div>
+              <div><p class="back-end">Back End Dev</p></div>
+              <div><img src="images/Counter.png" alt=""></div>
+              <div><p class="back-end">2015</p></div>
+          </div>
+          <div class="access-project">
+              <p>A daily selection of privately personalized reads. no accounts or sign-ups required</p>
+          </div>
+          <div class="programming-lang">
+              <div class="lang"><p>html</p></div>
+              <div class="lang"><p>css</p></div>
+              <div class="lang"><p>javascript</p></div>
+          </div>
+          <div class="project-btn"><button>See project</button></div>
+      </div>
+      </div>
+  </section>
+  <section>
+      <div class="main-container">
+        <div class="multiPostStories-con">
+          <div><img src="images/Snapshoot Portfolio-2.png" alt=""></div>
+          <div class="project-review">
+              <label>Multi Post Stories</label>
+          </div>
+          <div class="backend-dev">
+              <div><p class="canopy">Canopy</p></div>
+              <div><img src="images/Counter.png" alt=""></div>
+              <div><p class="back-end">Back End Dev</p></div>
+              <div><img src="images/Counter.png" alt=""></div>
+              <div><p class="back-end">2015</p></div>
+          </div>
+          <div class="access-project">
+              <p>A daily selection of privately personalized reads. no accounts or sign-ups required</p>
+          </div>
+          <div class="programming-lang">
+            <div class="lang"><p>html</p></div>
+            <div class="lang"><p>css</p></div>
+            <div class="lang"><p>javascript</p></div>
+          </div>
+          <div class="project-btn"><button>See project</button></div>
```

Comment:
- [ ] [OPTIONAL] It would be a great thing to give your buttons some interactivity so that they might be more intuitive when someone clicks on it they know that it is a button. If you might not be knowing how to implement this, here is an [awesome resource](https://zellwk.com/blog/style-hover-focus-active-states/) I found for you.

**Below I have included the button states as found from the chosen Figma design:**

![image](https://user-images.githubusercontent.com/86318284/192099832-f34328e8-ae77-4d28-97b4-7b7e6a4d6d1e.png)


**Note:** Even though this was not requested of you by the previous reviewer, it is a good thing to implement it before you merge it. In future projects, other reviewers will still require you to include the button states hence it is a good thing to have them there already.

**Kindly do so for all the buttons in the project**

Reply:
Thank you